docs/management_api.yml

```diff
@@ -28,83 +28,80 @@ tags:
   - name: ManagementAPI
 
 paths:
-  /configuration:
+  /config/devices/{deviceId}:
     get:
-      operationId: List Parameters
+      operationId: Get key-value pair store
       tags:
         - ManagementAPI
-      summary: Get existing parameters
-      # parameters: {}
+      summary: Query the configuration store; retrieve all key-value pairs
+      parameters:
+        - in: path
+          name: deviceId
+          schema:
+            type: string
+          required: true
+          description: ID of the device to query.
       responses:
         200:
           description: OK
           content:
             application/json:
               schema:
-                type: array
-                items:
-                  $ref: '#/components/schemas/Parameter'
+                type: object
+                properties:
+                  id:
+                    type: string
+                  expected:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/Parameter'
+                  actual:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/Parameter'
+                  updated_ts:
+                    type: string
+                  report_ts:
```

Comment:
To match `updated_ts`, I would suggest `reported_ts`.

Reply:
I removed all attributes, and the GET endpoint returns only the key-value pairs and not the whole device.